osu.Game/Screens/Play/PauseOverlay.cs

```diff
@@ -16,14 +19,40 @@ public class PauseOverlay : GameplayMenuOverlay
         public override string Header => "paused";
         public override string Description => "you're not going to do what i think you're going to do, are ya?";
 
+        private DrawableSample pauseLoop;
+
         protected override Action BackAction => () => InternalButtons.Children.First().Click();
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colours)
+        private void load(OsuColour colours, AudioManager audio)
         {
             AddButton("Continue", colours.Green, () => OnResume?.Invoke());
             AddButton("Retry", colours.YellowDark, () => OnRetry?.Invoke());
             AddButton("Quit", new Color4(170, 27, 39, 255), () => OnQuit?.Invoke());
+
+            var sampleChannel = audio.Samples.Get(@"Gameplay/pause-loop");
+
+            if (sampleChannel != null)
+            {
+                pauseLoop = new DrawableSample(sampleChannel)
+                {
+                    Looping = true,
+                };
+                AddInternal(pauseLoop);
+                pauseLoop?.Play();
```

Comment:
Not quite sure about always playing this, is there a problem with playing and stopping it in the below methods?

Reply:
The only reason was that it felt more natural to me, since the volume change already "stops" the sound for the user.

Reply:
I guess that's fine.